app/views/users/show.html.erb

```diff
@@ -1,5 +1,46 @@
-<h1>Username</h1>
-<ul>
-   <li><%= @user.name %></li>
-   <li><%= @user.bio %></li>
-</ul>
+<html>
+  <head>
+    <meta http-equiv = "Content-Type" content = "text/html; charset = iso-8859-1" />
+    <meta http-equiv = "Content-Language" content = "en-us" />
+    <title>Library Info System</title>
+    <%= stylesheet_link_tag "style" %>
+  </head>
+  <body>
+    <h1>Username</h1>
+    <table>
+      <tr>
+        <th>
+          <h1>ID</h1>
+        </th>
+        <th>
+          <h1>Users</h1>
+        </th>
+      </tr>
+      <tr>
+        <td>
+          <h2><%= @user.id %></h2>
+        </td>
+        <td>
+          <h2><%= @user.name %></h2>
+          Number of posts: <%= @user.posts_counter %>
+        </td>
+      </tr>
+    </table>
+    <div class="bio">
+      <h3>Bio</h3>
+      <p><%= @user.bio %></p>
+    </div>
+    <% @posts.each do |post| %>
+      <div class="bio bio2">
+        <div class="box1">
```

Comment:
- [ ] Please make sure to use the methods that you have created in the models (showing only the three most recent posts), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement.